CHANGELOG.md

```diff
@@ -4,6 +4,61 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.15.0] - 2023-04-17
+
+## 🚀 Features
+
+### GraphOS Enterprise: Allow JWT algorithm restrictions ([Issue #2714](https://github.com/apollographql/router/issues/2714))
+
+It is now possible to restrict the list of accepted algorthms to a well-known set for cases where an issuer's JSON Web Key Set (JWKS) contains keys which are usable with multiple algorithms.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2852
+
+## 🐛 Fixes
+
+### Invalid requests now return proper GraphQL-shaped errors ([Issue #2934](https://github.com/apollographql/router/issues/2934)), ([Issue #2946](https://github.com/apollographql/router/issues/2946))
+
+Certain invalid HTTP requests — such as when an unacceptable `content-type` header, or an unsupported `accept` header are received — now return proper GraphQL errors nested as elements in a top-level `errors` array, rather than returning a single GraphQL error JSON object, which was unintentional.
+
+This also introduced a more semantically-correct error code, `INVALID_CONTENT_TYPE_HEADER`, rather than using `INVALID_ACCEPT_HEADER` when an invalid `content-type` header was received.
+
+By [@EverlastingBugstopper](https://github.com/EverlastingBugstopper) in https://github.com/apollographql/router/pull/2947
+
+## 🛠 Maintenance
+
+### Remove redundant `println!()` that broke json formatted logging ([PR #2923](https://github.com/apollographql/router/pull/2923))
+
+The `println!()` statement being used in our trace transmission logic was redundant since it was already covered by a pre-existing `WARN` log line.  Most disruptively though, it broken JSON logging.
```

Comment:
```suggestion
The `println!()` statement being used in our trace transmission logic was redundant since it was already covered by a pre-existing `WARN` log line.  Most disruptively though, it broke JSON logging.
```